portal/config/config.py

```diff
@@ -229,6 +229,11 @@ class BaseConfig(object):
         'uwcirg.github.io',
     ]
 
+    BANNER_LOGOS = [
```

Comment:
Will there ever be more than 2 images?

Would it make sense to name the images based on the `PERSISTENCE_DIR` config name (ie `gil` or `eproms`)? (so we wouldn't need an extra config)